Sources/Fuzzilli/Evaluation/ProgramAspects.swift

```diff
@@ -16,11 +16,16 @@
 public class ProgramAspects: CustomStringConvertible {
     let outcome: ExecutionOutcome
     
-    init(outcome: ExecutionOutcome) {
+    public init(outcome: ExecutionOutcome) {
         self.outcome = outcome
     }
 
     public var description: String {
         return "execution outcome \(outcome)"
     }
+
+    // Produces the intersection of the interesting parts of each programs' aspects
```

Comment:
Does this method need to exist on `ProgramAspects`? It's probably only useful on `CovEdgeSet`, so if possible, I'd remove it from this class